source/navitiacommon/navitiacommon/models.py

```diff
@@ -432,6 +432,37 @@ def delete_dataset(self):
 
         db.session.commit()
 
+    def delete_old_jobs(self, time_limit):
```

Comment:
minor: not fan of the naming, maybe `list_and_delete_old_jobs`

Reply:
To be discussed

Reply:
As you wish :)